Makefile

```diff
@@ -0,0 +1,43 @@
+
+# Destroys your kind cluster completely
+kinddown:
```

Comment:
Just my 2c on this: we should scope a bit `bazel` usage to just build, and not try to make it a deploy platform. Feel free to ignore this comment :)